scripts/travis-extract-l10n.sh

```diff
@@ -40,6 +51,20 @@ function init_environment {
 }
 
 
+function restore_environment {
+    echo "Restoring global git configuration"
+
+    git config --global user.email "$GIT_USER_EMAIL"
+    git config --global user.name "$GIT_USER_NAME"
+
+    if [ "$GIT_SIGNING" == "true" ]; then
+        git config --global commit.gpgsign "$(GIT_SIGNING)"
+        git config --global user.signinkey "$(GIT_SIGNING_KEY)"
```

Comment:
Not sure what's going on here - `init_environment` above doesn't seem to set signing config but `restore_environment` restores them.

Reply:
This is for everyone who runs this script locally (as I do) and uses git signing, since the `addons-robot` doesn't have a proper gpg key as of now we can't sign the commits and have to disable that config in case someone else does, because a email mismatch results in an error rather than being ignored.